[2025-03-01T01:40:28.672+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-01T01:40:28.690+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_postgres_1.generate_and_insert_orders manual__2025-03-01T01:40:23.331054+00:00 [queued]>
[2025-03-01T01:40:28.697+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_postgres_1.generate_and_insert_orders manual__2025-03-01T01:40:23.331054+00:00 [queued]>
[2025-03-01T01:40:28.698+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-01T01:40:28.710+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): generate_and_insert_orders> on 2025-03-01 01:40:23.331054+00:00
[2025-03-01T01:40:28.720+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=41574) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-01T01:40:28.720+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_postgres_1', 'generate_and_insert_orders', 'manual__2025-03-01T01:40:23.331054+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/dag_generate_table_info.py', '--cfg-path', '/tmp/tmp627bcbv6']
[2025-03-01T01:40:28.721+0000] {standard_task_runner.py:72} INFO - Started process 41580 to run task
[2025-03-01T01:40:28.721+0000] {standard_task_runner.py:105} INFO - Job 49: Subtask generate_and_insert_orders
[2025-03-01T01:40:28.759+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_postgres_1.generate_and_insert_orders manual__2025-03-01T01:40:23.331054+00:00 [running]> on host e6b43e607126
[2025-03-01T01:40:28.829+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kucikm23@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_postgres_1' AIRFLOW_CTX_TASK_ID='generate_and_insert_orders' AIRFLOW_CTX_EXECUTION_DATE='2025-03-01T01:40:23.331054+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-01T01:40:23.331054+00:00'
[2025-03-01T01:40:28.830+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-01T01:40:28.831+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-01T01:40:28.831+0000] {logging_mixin.py:190} INFO - Current task name:generate_and_insert_orders state:running start_date:2025-03-01 01:40:28.690874+00:00
[2025-03-01T01:40:28.831+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 and current dag run status:running
[2025-03-01T01:40:28.831+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-01T01:40:28.929+0000] {base.py:84} INFO - Retrieving connection 'postgres_conn'
[2025-03-01T01:40:33.361+0000] {sql.py:639} INFO - Loaded 5000 rows into orders so far
[2025-03-01T01:40:37.964+0000] {sql.py:639} INFO - Loaded 5000 rows into orders so far
[2025-03-01T01:40:37.964+0000] {sql.py:652} INFO - Done loading. Loaded a total of 10000 rows into orders
[2025-03-01T01:40:37.966+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-01T01:40:37.975+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-01T01:40:37.976+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_postgres_1, task_id=generate_and_insert_orders, run_id=manual__2025-03-01T01:40:23.331054+00:00, execution_date=20250301T014023, start_date=20250301T014028, end_date=20250301T014037
[2025-03-01T01:40:37.999+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-01T01:40:38.000+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-01T01:40:38.000+0000] {logging_mixin.py:190} INFO - Dag name:dag_postgres_1 queued_at:2025-03-01 01:40:23.339408+00:00
[2025-03-01T01:40:38.001+0000] {logging_mixin.py:190} INFO - Task hostname:e6b43e607126 operator:PythonOperator
[2025-03-01T01:40:38.037+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-01T01:40:38.052+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-01T01:40:38.055+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
